controller/src/enhancements/player.rs

```diff
@@ -346,6 +346,59 @@ impl Enhancement for PlayerESP {
                             draw.add_rect([vmin.x, vmin.y], [vmax.x, vmax.y], *esp_color)
                                 .thickness(settings.esp_boxes_thickness)
                                 .build();
+
+                            if settings.esp_health_bar {
+                                let bar_height = vmax.y - vmin.y; // height = box height
+                                let bar_x = vmin.x - 5.0; // Left
+
+                                let max_health = 100.0;
+                                let player_health = entry.player_health;
+                                let health_percentage = player_health as f32 / max_health as f32;
```

Comment:
You need to ensure, that `player_health` never exceeds `HEALTH_BAR_MAX_HEALTH`.  
Best would be to clamp `player_health` between `0.0` and `HEALTH_BAR_MAX_HEALTH` to avoid any unexpected `health_percentage` values.